setup.cfg

```diff
@@ -28,13 +28,7 @@ url = https://github.com/cta-observatory/ctapipe
 edit_on_github = False
 github_project = cta-observatory/ctapipe
 
-[pep8]
-exclude=astropy_helpers,ah_bootstrap.py,ez_setup.py,build,examples/brainstorm,docs
-max-line-length=90
-ignore=W291,E303,W391,F403,F401,W503,W1202,E402 
-
-
-[flake8]
+[pycodestyle]
 exclude=
   astropy_helpers,
```

Comment:
These files all got removed from the repo

Reply:
done.